### ISSUE TITLE
Product rating and number of reviews not vocalized

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13576]
**Describe the bug**

- Product rating is not being narrated properly. Currently not receiving keyboard focus and this is OK, but if the user manages to get there by "Read Mode", is also unable to get there as it is set as "disabled".
- The number of reviews doesn't get narrated as such, but only as a number. Currently not receiving keyboard focus and this is OK, but if the user manages to get there by "Read Mode", it only gets vocalized as "3, text element".

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS)
2. Go to a Product Details page with reviews (E.g. HDR-CX105E Red)
3. Tab to "Show Reviews" Link
4. Switch to "Read Mode" in Screen Reader (navigate with aux key + arrows instead of Tab key).
5. Listen to SR narration.

**Expected behavior**
- When browsing in "read mode" Screen Reader vocalizes rating as: _"Rated 3.5 out of 5 stars"_
- When browsing in "read mode" Screen Reader vocalizes the number of reviews as: _"19 reviews"_
- When focusing on the "Show reviews" button Screen Reader vocalizes: _"Show reviews, button, 19 reviews, Rated 3.5 out of 5 stars"_
- Use `aria-label`, `aria-describedby`, `cx-visually-hidden`

**Screenshots**
![Screen Shot 2021-08-30 at 7 56 58 PM](https://user-images.githubusercontent.com/15113219/131420568-08178480-2755-4b10-9ee2-a0dacb66d748.png)

**Potential Solution**
(ID names are only examples, please follow standard naming convention for IDs.)

![Screen Shot 2021-08-31 at 8 02 19 AM](https://user-images.githubusercontent.com/15113219/131500886-a85b2019-f809-4913-bb61-aa3250a6daa9.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

@developpeurweb Can you confirm this is what you want?
Ratings does not read anything. 
Number of reviews just read the number. 

<img width="1279" alt="Screen Shot 2021-12-09 at 11 36 30 AM" src="https://user-images.githubusercontent.com/45409515/145438203-d47fb10b-4c5f-40bb-be51-0e5865f4d370.png">


"Show Reviews" when is in focus and in read mode: 
![productReviews](https://user-images.githubusercontent.com/45409515/145436762-4d08f920-e5bb-4b63-be19-97a25612e555.png)

**Comment by developpeurweb** [comment]
@KateChuen yes, the number of reviews and the rating when focusing **is a good outcome**.
(I believe the other 2 for read-mode were not possible, but that is OK, as long as we have the aforementioned.)
